### PR TITLE
Migration to Typescript

### PR DESCRIPTION
**Pull request by endway**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -46,14 +46,19 @@
   "tabs",
   "storage",
   "notifications",
-  "http://vk.com/*"
+  "http://vk.com/*",
+  "https://play.google.com/music/*",
```
Comment (beshur) [inline manifest.json]: why are these here?
"tabs" seem suffice.

File: src/content/services/base.service.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+		// const checkTitleInterval = setInterval(() => this.checkTitle(), 10000);
+	}
+
+	public playingTitle(): string {
+		return "";
+	}
+
+	private checkTitle() {
+		// TODO: fix dis shit
```
Comment (beshur) [inline src/content/services/base.service.ts]: so this is not only moving to TS

File: manifest.json (hunk 1 above)
Comment (endway) [reply manifest.json]: oops

File: src/content/services/base.service.ts (hunk 2 above)
Comment (endway) [reply src/content/services/base.service.ts]: we already have such a method
https://github.com/StoPlay/stoplay-ext/blob/d2cf04cf168d4b2cf1ec8065929e54200266d72e/content.js#L145

**Comment by endway** [comment]
seems like this PR will never be finished
